spec/models/event_spec.rb

```diff
@@ -25,5 +24,19 @@
       expect(event).to be_invalid
       expect(event.errors.messages.keys).to include :title
     end
+
+    it "does not validate when 'published_by' is not defined" do
+      event = Event.new(published_by: nil)
+      expect(event).to be_invalid
+      expect(event.errors.messages.keys).to include :published_by
+    end
+  end
+
+  context 'when an event is published' do
+    it 'knows about its publisher' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.